compiler-rt/lib/interception/tests/interception_win_test.cpp

```diff
@@ -230,6 +230,7 @@ const u8 kUnpatchableCode6[] = {
     0x90, 0x90, 0x90, 0x90,
 };
 
+#if SANITIZER_WINDOWS64
```

Comment:
Why these new `#if`s here (same for the others below)? So far, these tests pass fine on both i686 and x86_64, see e.g. https://github.com/mstorsjo/llvm-mingw/actions/runs/11883858793/job/33120364287.

Reply:
Ah, I see that this gets rid of warnings like this: `warning: unused variable 'kUnpatchableCode7' [-Wunused-const-variable]`

That's probably fine, but it would distract less from the rest of this MR if it would be filed as a separate cleanup.